HousingSearchApi/V1/Interfaces/AssetQueryGenerator.cs

```diff
@@ -16,18 +19,35 @@ public AssetQueryGenerator(IWildCardAppenderAndPrepender wildCardAppenderAndPrep
 
         public QueryContainer Create(HousingSearchRequest request, QueryContainerDescriptor<QueryableAsset> q)
         {
-            if (string.IsNullOrWhiteSpace(request.SearchText)) return null;
```

Comment:
Surely you still need to do this? There should always be some search text shouldn't there?

Reply:
if requests without search text are never valid, would it make sense to instead have the `[Required]` attribute on the Request model?

Reply:
> if requests without search text are never valid, would it make sense to instead have the `[Required]` attribute on the Request model?

I've just noticed we are using `FluentValidator` so a request with empty search text will never even make it to the controller

Reply:
It would still be good to do defensive coding. Check where we can. :)